lib/AST/ASTImporter.cpp

```diff
@@ -1604,12 +1657,12 @@ Decl *ASTNodeImporter::VisitAccessSpecDecl(AccessSpecDecl *D) {
   if (!DC)
     return nullptr;
 
-  AccessSpecDecl *accessSpecDecl
-    = AccessSpecDecl::Create(Importer.getToContext(), D->getAccess(),
-                             DC, Loc, ColonLoc);
-
-  if (!accessSpecDecl)
-    return nullptr;
+  AccessSpecDecl *accessSpecDecl;
+  bool NewlyCreated;
+  std::tie(accessSpecDecl, NewlyCreated) = CreateDecl<AccessSpecDecl>(
+      D, Importer.getToContext(), D->getAccess(), DC, Loc, ColonLoc);
+  if (!NewlyCreated)
+    return accessSpecDecl;
```

Comment:
We no longer handle the case where create returns nullptr, is this intentional?

Reply:
nullptr should never returned by Create function.